src/Agent.Listener/DistributedTask.Pipelines/Yaml/Contracts/PhasesTemplate.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.TeamFoundation.DistributedTask.Orchestration.Server.Pipelines.Yaml.Contracts
+{
+    // A phase template cannot reference other phase templates, but
+    // steps within can reference templates.
```

Comment:
this isnt intellisense comments, this is just author notes for anyone who needs to maintain the source code